css/style.css

```diff
@@ -184,12 +227,119 @@ footer h3  {
 }
 
 
-
 footer  {
     
     left: 0;
     bottom: 0;
     width: 100%;
     text-align: center;
+}
+/* css for store.html */
+
+.rightcontent {
+    width: 50% ;
+    height:auto;
+}
+ 
+.secondpic {
+    width:100%;
+    height:auto;   
+}
+
+.content {
+    display: flex;
+    flex-direction: row-reverse;
+    
+}
+
+.title {
+    font-family: Roboto;
+    font-style: normal;
+    font-size: 30px;
+    color: #f36100f1;
+    font-weight: 900;
+/* border:2 px solid black; */
+
+}
+
+.leftcontent {
+    font-family: Roboto;
+    font-style: normal;
+    font-size: 15px;
+    color: #666666;
+    padding-left: 30px;
+    width: 50%;
+    height: auto;
+    padding-right: 40px;
+}
+
+fieldset {
+    border: 0;
+}
+
+.details {
+    display: flex;
+    justify-content: space-between;
+    
+}
+
+
+.address {
+    display: flex;
+    flex-direction: column;
+}
+
+
+
+.checkboxterms a {
+    color:#f36100f1;
+}
+
+ 
+.details input {
+    width: 235px;
+    height: 38px;
+    margin-top:11px;
+    border-radius: 4px;
+    border: 2px solid #666666;
+}
+/* for city form */
+
+.details select {
+    width: 235px;
+    height: 45px;
+    margin-top:11px;
+    border-radius: 4px;
+    border: 2px solid #666666;
+}
+
+.address input {
+    width: 515px;
+    height: 38px;
+    margin-top:11px;
+    border-radius: 4px;
+    border: 2px solid #666666;
+}
+
+.radio_box {
+    margin: 45px 0px;
+    /* display: inline-flex; */
+    
+
+}
+.radio_box label{
+    margin-right: 50px;
+   margin-left: 7px;
+   
+}
+
+.radio_box input {
```

Comment:
`vertical-align: sub;` or `vertical-align: middle;` to put the text inline with radio button.